src/html/assets/themes/fabrizio1.0/templates/partials/header-secondary.php

```diff
@@ -17,10 +17,6 @@
                 <li><a href="#" class="triggerShare"><i class="icon-share-alt triggerShare"></i> Share this album</a></li>
               <?php } ?>
             <?php } ?>
-          <?php } elseif($this->utility->isActiveTab('upload')) {?>
```

Comment:
We should be able to keep these links in place. The official Trovebox app was removed but there is [another iOS app](https://github.com/delightfulapp/delightful) by @delightfulapp. The iTunes link is here, https://itunes.apple.com/us/app/delightful-trovebox-photo/id878230610?mt=8

Is there an APK we can upload somewhere so folks can install the Android build? @patricksan